keras/layers/wrappers.py

```diff
@@ -114,7 +114,7 @@ def step(x, states):
                 return output, []
 
             last_output, outputs, states = K.rnn(step, X,
-                                                 initial_states=[])
+                                                 initial_states=[], unroll=K._BACKEND == 'tensorflow')
```

Comment:
The correct way to access the current backend is `K.backend() == `

Better to set an `unroll` variable outside.

With `unroll=True` we need to make sure that the number of timesteps is defined, so we will need a check and exception in that case.